executor/execute_function.go

```diff
@@ -15,20 +15,18 @@ func (e *Executor) ExecuteFunction(requestID string, req execute.Request) (execu
 	if err != nil {
 
 		res := execute.Result{
-			Code:      codes.Error,
-			RequestID: requestID,
```

Comment:
unsure why we're dropping requestId here?

Reply:
Request ID is a repeated field in the models, so it's not really needed here. I wanted to trim this struct to only contain data that stems from the execution itself.

The request ID is still in the `response.Execute` type that we send. So this is just deduplication.